deps/meck-0.7.2/src/meck.erl

```diff
@@ -271,7 +269,7 @@ history(Mod) when is_atom(Mod) -> call(Mod, history).
 %% @see num_calls/3
 %% @see num_calls/4
 -spec history(Mod::atom(), Pid:: pid() | '_') -> history().
-history(Mod, Pid) when is_atom(Mod), is_pid(Pid) orelse Pid == '_' -> 
```

Comment:
Emacs does whitespace removal when I save any file. I didn't do this, or the others, manually.